packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -624,7 +628,8 @@ class Badge extends StatelessWidget {
       child: Text(
         text,
         // Use a slightly smaller font for the badge.
-        style: theme.primaryTextTheme.bodyText2.apply(fontSizeDelta: -1),
+        style: (theme.primaryTextTheme.bodyText2 ?? const TextStyle())
```

Comment:
can we do `theme.primaryTextTheme.bodyText2!` here instead of adding `?? const TextStyle()`

Reply:
what if theme.textTheme.bodyText2 is null?

Reply:
Hmm. I think this value is set by default by the flutter framework, so we've always treated it as non-nullable in our code. But I see that all the values of TextTheme are nullable, so it's fine for us to be consistent here.